src/components/summary.json

```diff
@@ -174,11 +174,11 @@
            {
              "headers": {
                "Drug Description": "Name",
-               "Quantity": { "key": "Quantity", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": false},
+               "Quantity": { "key": "Quantity", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": true},
```

Comment:
same here, if we have the field information, we **probably** don't need `stringSubstitutionFormat` formatter here:
`{ "key": "Quantity", "sortable": true}`

Reply:
noting here that I did need to change it to this:  `"Quantity": { "key": "Quantity", "sortable": true}` for the quantity to show